tag_utils/dlrn.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import json
+
+import requests
+
+
+from .common import tidy_nevra
+
+
+def fetch_json_data(url):
+    ret = requests.get(url)
+    if ret.status_code == 200:
+        return json.loads(ret.text)
+
+
+def json_api_as_nevr(url):
+    md = fetch_json_data(url)
```

Comment:
question: can we use a more descriptive variable name here? Obviously 'md' is going to be  the returned json, but maybe we could just spell out what this abbreviation is for, so it is more clear to readers?